packages/astro/src/core/errors/errors-data.ts

```diff
@@ -1001,6 +1001,13 @@ export const MissingLocale = {
 		`The locale/path \`${locale}\` does not exist in the configured \`i18n.locales\`.`,
 } satisfies ErrorData;
 
+export const MissingIndexForInternationalization = {
+	name: 'MissingIndexForInternationalizationError',
+	title: 'Index URL not found.',
```

Comment:
```suggestion
	title: 'Index page not found.',
```
Since it's the index, it's always `/` so maybe saying "page" is less confusing?